### PR TITLE
Merge branch to master

### PR DESCRIPTION
**Pull request by tjhcb**
Hi..
I would like to push my ideas to your master so that they could be considered for implementation.
Please review and provide your inputs.

### CODE REVIEW

**Comment by tejaswiniwp** [comment]
Let me verify and wait for other members to join in the review.

**Comment by tejaswiniwp** [comment]
We are fine with changes.. Please go on with merge. :thumbsup:

**Comment by tjhcb** [comment]
Thanks.. Merging changes to master :simple_smile:

**Comment by tjhcb** [comment]
Thanks.. Merging changes to master... :relieved:

File: ChatRoom.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Hi there... 
+I would like to bring in focus the idea of platform independent Chat Room for colleagues in offices.
```
Comment (tjhcb) [inline ChatRoom.txt:2]: This idea is worth implementing but should add some different features

**Pull request merged**